modules/serverless-interacting-apps-HTTPS.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * serverless/interacting-serverless-apps.adoc
+
+[id="serverless-interacting-apps-HTTPS_{context}"]
```

Comment:
I don't know if it's a rule, but typically we'd keep the anchor text all lowercase, even for an all caps item like HTTPS.